sematic/ui/packages/common/src/utils/datetime.ts

```diff
@@ -0,0 +1,27 @@
+function _commonConvertor(displayH: number, displayM: number, displayS: number) {
+    const final = [
+        displayH > 0 ? displayH.toString() + "h" : "",
+        displayM > 0 ? displayM.toString() + "m " : "",
+        displayS > 0 ? displayS.toString() + "s" : "",
+        displayS === 0 ? "<1s" : "",
```

Comment:
Shouldn't `"<1s"` only be displayed when all 3 parameters are 0?

Reply:
Thanks. I am simply migrating code; now I am confused too. Can @neutralino1 help confirm if there is a special purpose for doing it that way? If not, I can fix it.

Reply:
I had a fix for that now.